leasing/report/invoice/invoice_payments.py

```diff
@@ -0,0 +1,76 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.response import Response
+
+from leasing.models.invoice import InvoicePayment
+from leasing.report.excel import ExcelCell, ExcelRow, SumCell
+from leasing.report.report_base import ReportBase
+
+
+def get_invoice_number(obj):
+    return obj.invoice.number
+
+
+def get_lease_id(obj):
```

Comment:
I guess these helper functions could be refactored into their own module to be imported from since at least `get_lease_id` is duplicated across different report modules but OTOH I sort of like that the reports are self contained in that they define their own helpers.
I think we will just leave it as is.

Reply:
I was thinking exactly that the reports would be mostly self contained. Also, if you look closely, you'll see that at other places the lease could be read from obj.lease, here it's obj.invoice.lease, etc.